src/components/Item/DiscountPrice.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+import { colorRed } from '../../styles/designTokens';
+import { RegularPrice } from './Price';
+
+const Wrapper = styled(RegularPrice)`
+  text-transform: uppercase;
+  color: ${colorRed};
+  font-weight: bold;
+`;
+
+function DiscountPrice(props) {
```

Comment:
I think you are taking it a bit too far here. It's enough to leave Wrapper as DiscountPrice and use it directly in Item.
Like (assuming item comes from redux):
```
function Item({item}) {
  return (
    ...
    <DiscountPrice>Sale ${item.price}</DiscountPrice>
    ...
  );
}
```